backend/tests/prisonerLocationHistory.test.js

```diff
@@ -5,6 +5,8 @@ describe('Prisoner location sharing history', () => {
   const offenderNo = 'ABC123'
```

Comment:
Missing unit tests to cover the logic that generates: **No staff name found** and the other custom error wrappers.

Reply:
there is a test for whatHappened will do the other two as well

Reply:
more tests added